packages/palette/src/elements/Modal/Modal.tsx

```diff
@@ -86,6 +87,7 @@ export const Modal: SFC<ModalProps> = ({
 
   useEffect(() => {
     if (show) {
+      setRenderModal(true)
```

Comment:
If this effect depends on `show` (L118) and our return value looks like ```
<>{renderModal && (<ModalOuterWrapper show={show}> ...
```

Does that mean that the `ModalOuterWrapper` effects achieved by its `show` prop would be lost (since the modal would simply no longer render after `useEffect` first with `show=false`?

Reply:
Hey it's a bit hard to read/get what you're saying here, maybe we can review in person?

Reply:
Talked in person, verified on deploy preview, all good.